Sources/SmithyChecksums/ChunkedReader.swift

```diff
@@ -114,11 +114,6 @@ public class ChunkedReader {
 
```

Comment:
This early exit for unsigned chunk was quitting early before entirety of body is streamed, making S3 return `IncompleteBody` error.